core/iwasm/common/wasm_shared_memory.c

```diff
@@ -307,6 +308,8 @@ acquire_wait_info(void *address, bool create)
 
     if (address)
         wait_info = (AtomicWaitInfo *)bh_hash_map_find(wait_map, address);
+    if (wait_info)
+        ++wait_info->count_acquisition;
```

Comment:
Here we're accessing `count_acquisition` without a lock on `wait_info`

Reply:
Fair point, I'll try to reorder stuff to use the existing list then

Reply:
Just remembered it's like that because count_acquisition memory accessed only under wait_map_lock. But I will try to change to using the existing list